util/perl/OpenSSL/Ordinals.pm

```diff
@@ -757,8 +772,8 @@ STRING must conform to the following EBNF description:
 
 (C<letter> and C<digit> are assumed self evident)
 
-=item B<< name => STRING >>, B<< number => NUMBER >>, B<< version => STRING >>,
-      B<< exists => BOOLEAN >>, B<< type => STRING >>,
+=item B<< source => STRING >>, B<< name => STRING >>, B<< number => NUMBER >>,
```

Comment:
For consistency, shouldn't that be `B<< source => FILENAME >>` ?

Reply:
Changed